spec/factories/cart.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :cart do
```

Comment:
why empty?

Reply:
Because cart has only created_at, updated_at and id attributes, which should be created automatically. But, factory cart_item has an association ‘belongs_to’ with cart, so it needs a reference to the cart factory.